web-common/src/features/feature-flags.ts

```diff
@@ -30,6 +30,7 @@ type FeatureFlagKey = keyof Omit<FeatureFlags, "set">;
 class FeatureFlags {
   adminServer = new FeatureFlag("rill", false);
   readOnly = new FeatureFlag("rill", false);
+  sourceImportedModal = new FeatureFlag("rill", true);
```

Comment:
We should not use `FeatureFlags` for this. Lets have a separate set of values for such use cases.